kingpin/actors/rightscale/deployment.py

```diff
@@ -98,13 +107,43 @@ def _execute(self):
             self.log.info('Would create a deployment %s' % self.option('name'))
             self.log.debug('Deployment params: %s' % params)
             raise gen.Return()
-        self.log.info('Deployment params: %s' % params)
 
         try:
             yield self._client.create_resource(
                 self._client._client.deployments, params)
         except Exception as e:
```

Comment:
I'd have to catch `requests.http.exceptions` and would import `requests` just for that... that's ok?

Any way to configure `rightscale.settings.RETRYING_SETTINGS` similar to our tornado retry? To raise a specified exception for a matched string?